lms/djangoapps/courseware/management/commands/dump_to_neo4j.py

```diff
@@ -0,0 +1,207 @@
+"""
+This file contains a management command for exporting the modulestore to
+neo4j, a graph database.
+"""
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from py2neo import Graph, Node, Relationship, authenticate
+from py2neo.compat import integer, string, unicode as neo4j_unicode
+from request_cache.middleware import RequestCache
+from xmodule.modulestore.django import modulestore
+
+log = logging.getLogger(__name__)
+
+# When testing locally, neo4j's bolt logger was noisy, so we'll only have it
+# emit logs if there's an error.
+bolt_log = logging.getLogger('neo4j.bolt')  # pylint: disable=invalid-name
+bolt_log.setLevel(logging.ERROR)
+
+ITERABLE_NEO4J_TYPES = (tuple, list, set, frozenset)
+PRIMITIVE_NEO4J_TYPES = (integer, string, neo4j_unicode, float, bool)
+
+
+class ModuleStoreSerializer(object):
+    """
+    Class with functionality to serialize a modulestore into subgraphs,
+    one graph per course.
+    """
+    def __init__(self):
+        self.all_courses = modulestore().get_course_summaries()
+
+    @staticmethod
+    def serialize_item(item):
+        """
+        Args:
+            item: an XBlock
+
+        Returns:
+            fields: a dictionary of an XBlock's field names and values
+            label: the name of the XBlock's type (i.e. 'course'
+            or 'problem')
+        """
+        # convert all fields to a dict and filter out parent and children field
+        fields = dict(
+            (field, field_value.read_from(item))
+            for (field, field_value) in item.fields.iteritems()
```

Comment:
I think the outcome of the HipChat conversation was that six is currently an indirect dependency pulled in by other packages, but is being added as an explicit dependency in another PR.  Could use django.utils.six in the meantime to be safe.  And yes, `unicode_literals` and `absolute_import` are usually good things to import.